modules/ROOT/pages/shared-resources.adoc

```diff
@@ -107,14 +106,29 @@ You can configure the <<create_domain_project,Domain Project>> by defining share
 You can add module or connector dependencies to you domain project so you can use them as shared resources.
 
 [[assoc_apps_domain]]
-=== Associating Apps with the Domain
+== Associating Applications with a Domain
+
+Apps can only be associated with one domain at a time. You can associate an existing app with a domain using Studio, or by editing your project's `pom.xml` file directly. If you don't set any domain dependency, applications use the `default` domain.
+
+Consider the following when working with Mule 4.2.2 and later versions:
 
-Apps can only be associated with one domain at a time. To associate an existing app with a domain from Studio:
+* The version honors semantic versioning. For example, if you set the version to `1.0.1`, a domain with version `1.0.2` and later works, but a domain with version `1.0.0` does not.
```

Comment:
GLOBALLY limit each list item or step to one sentence or fragment max, with any necessary clarification or details in a separate para below the item or step:
```suggestion
* The version honors semantic versioning. 
For example, if you set the version to 1.0.1, a domain with version 1.0.2 and later works, but a domain with version 1.0.0 does not.
```
We don't need to format these as syntax values, because this is a general case rather than a precise syntax example, and the values make sense as such.